tests/PHPStan/Analyser/DynamicReturnTypeExtensionTypeInferenceTest.php

```diff
@@ -19,6 +19,7 @@ public function dataAsserts(): iterable
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/dynamic-method-return-compound-types.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/dynamic-method-return-getsingle-conditional.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-7344.php');
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-7391.php');
```

Comment:
Instead of moving and changing and existing test, please add a new test file instead.

Reply:
done